gazebo_msgs/srv/GetModelProperties.srv

```diff
@@ -1,9 +1,9 @@
 string model_name                    # name of Gazebo Model
```

Comment:
Can we change the message for `eloquent`? We will keep the same message for dashing.

Reply:
We still need the bridge to work for Eloquent. But here's what we can do:

* On Eloquent, have the fields side-by-side, accept both, but print warnings if the user is setting the old ones
* We probably don't want to change this for F turtle, since people will still be using the bridge for a long time. So we can keep this deprecated state for a few distros.

What do you think?

Reply:
SGTM. I have reverted the changes to message for now aac7178